src/Domain.jl

```diff
@@ -1790,6 +1790,23 @@ function jacobiany!(jac::Q,y::U,p::W,t::Z,domain::D,interfaces::Q3,colorvec::Q2=
     return jac
 end
 
+function jacobiany!(jac::Q,y::U,p::W,t::Z,domain::D,interfaces::Q3,colorvec::Q2=nothing) where {Q3<:AbstractArray,Q2,Q<:AbstractArray,U<:AbstractArray,W,Z<:Real,D<:AbstractDomain}
```

Comment:
What Domains is this function intended for? Is this capable of generalizing to arbitrary new domains someone might write into RMS?

Reply:
This is intended for ConstantTVDomain and ConstantTADomain. I will use Union{ConstantTVDomain, ConstantTADomain} instead.